lib/commands/screenbuilder.ts

```diff
@@ -0,0 +1,18 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+export class ScreenBuilderCommand implements ICommand {
+
+	constructor(private $logger: ILogger,
+		private $screenBuilderService: IScreenBuilderService) { }
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			var allSupportedCommands = this.$screenBuilderService.allSupportedCommands().wait();
+			_.each(allSupportedCommands, (command, index) => this.$logger.out("%s: %s", index.toString(), command));
+		}).future<void>()();
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+}
+$injector.registerCommand("screenbuilder", ScreenBuilderCommand);
```

Comment:
I couldn't figure out better name for this command, so I'm open to any suggestion.